test/letters/01-authed.e2e.spec.js

```diff
@@ -83,6 +83,43 @@ module.exports = E2eHelpers.createE2eTest(
       .click('div.step-content > p:nth-child(3) > a') // link to go back to confirm-address
       .expect.element('.city-state').to.be.present.before(Timeouts.normal);
 
+    // -- Go to letters list -- //
+
+    client
+      .click('.view-letters-button')
+      .assert.urlContains('/letters/letter-list')
+      .waitForElementVisible('.step-content', Timeouts.normal)
+      .click('.step-content div.form-review-panel:nth-of-type(4) button') // open the bsl accordion
```

Comment:
😬

Reply:
I considered dynamically adding an id to these accordion buttons, but it seems like more work than it was worth. I pretty much hate this selector, but it works. Until we change things. Then we cry. 😢